test/lib/api-v1.test.ts

```diff
@@ -226,6 +226,25 @@ describe("lib/api-v1", () => {
     });
   });
 
+  describe("getCommit", () => {
```

Comment:
Not on this PR, but following up on my [comment](https://github.com/knocklabs/control/pull/2598/files#r1389696396) from the control side PR: would you mind opening a PR and adding integration tests for the get endpoint if missing?

Reply:
I think these are the ones you are looking for right? https://github.com/knocklabs/control/blob/main/backend/test/control_web/controllers/v1/commit_controller/show_commit_test.exs

Reply:
Dohh, you are right we got it covered. Disregard 😅